package.json

```diff
@@ -6,6 +6,7 @@
   "type": "module",
   "scripts": {
     "clean": "rm -rf build && rm -rf __lwr_cache__",
+    "lint": "npx --yes prettier --write src",
```

Comment:
Linting on the fly. Could be wrapped into a pre-commit hook, but currently it's manual.